### PR TITLE
Update research.yml

### PR DESCRIPTION
**Pull request by Petess**
Thanks for the contribution! **Below** is a template that can make it easier when submitting your Organization:

**Your Organization**: _ACME Agency_  
**GitHub Organization url**: _@acmeagency_  
**Country/Locality**: _Alberta, Canada_

Inclusion requirements:
- [ ] Referenced account is an [Organization](https://github.com/github/government.github.com#add-organization) not a User^
- [ ] If this is a government project, your Organization's description should include a reference to your country/agency.
- [ ] Make sure your Organization includes at least one public repository
- [ ] Please also include a URL for your organization that links back to your organization or agency homepage.
 
^ If you want to convert your account to an org, details can be found here: https://help.github.com/articles/converting-a-user-into-an-organization/
  
 :heart: GitHub Government Contributors

### CODE REVIEW

File: _data/research.yml [hunk 1]
```diff
@@ -3,6 +3,12 @@
 International:
   - CICE-Consortium
 
+Australia:
+  - CSIRO-GeoscienceAnalytics
```
Comment (mlinksva) [inline _data/research.yml:5]: These should be alphabetized.

**Review by mlinksva — CHANGES_REQUESTED** [review]

File: _data/research.yml (hunk 1 above)
Comment (Petess) [reply _data/research.yml:5]: They are now in order.

File: _data/research.yml [hunk 2]
```diff
@@ -3,6 +3,12 @@
 International:
   - CICE-Consortium
 
+Australia:
```
Comment (nschonni) [inline _data/research.yml:3]: This node also needs to get put above "International"

Comment (nschonni) [reply _data/research.yml:3]: You can see the order it's expecting in the CI message https://github.com/github/government.github.com/pull/833/checks?check_run_id=1115294636#step:4:253

Comment (Petess) [reply _data/research.yml:3]: D'oh. Righto. Fixed now. Thanks for your patience.

**Review by Petess — COMMENTED** [review]
Fixed order of Australian research organisations again.

**Review by mlinksva — APPROVED** [review]

**Pull request merged**